.github/CODEOWNERS

```diff
@@ -14,12 +14,13 @@
 /angular-workspace/projects/ni/nimble-angular @rajsite @atmgrifter00
 /packages/nimble-blazor @atmgrifter00 @msmithNI
 /packages/nimble-components @rajsite @jattasNI
+/packages/nimble-components/.storybook @rajsite @jattasNI @fredvisser
 /packages/nimble-tokens @rajsite @jattasNI @fredvisser
 /packages/site @rajsite @jattasNI
 /packages/xliff-to-json-converter @rajsite @TrevorKarjanis @jattasNI
 
 # Specs (placed after packages to have higher precedence for spec files contained in packages)
-specs @rajsite @jattasNI @fredvisser @atmgrifter00
+specs @rajsite @jattasNI @fredvisser @atmgrifter00 @mollykreis
```

Comment:
Do we still think that @fredvisser should be added on all the specs?

Reply:
I'm curious if @fredvisser actually wants to be a CODEOWNER and pestered for reviews on changes to the `.storybook` config as well

Reply:
I don't need to be on the specs, but would be happy to review `.storybook` changes